### PR TITLE
LHCb VP2 pattern recognition GSOC proposal

### PR DESCRIPTION
**Pull request by roelaaij**

### CODE REVIEW

File: _gsocorgs/2019/nikhef.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Nikhef; Dutch national institute for subatomic physics"
+author: "Roel Aaij"
+layout: default
+organization: Nikhef
+logo: logo_Nikhef.jpg
+description: |
+[Nikhef](http://www.nikhef.nl) is the Dutch national institute for
```
Comment (aflp91) [inline _gsocorgs/2019/nikhef.md]: There are missing spaces in front of the sentence that prevent YAML reading.

File: _gsocorgs/2019/nikhef.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Nikhef; Dutch national institute for subatomic physics"
+author: "Roel Aaij"
+layout: default
+organization: Nikhef
+logo: logo_Nikhef.jpg
```
Comment (aflp91) [inline _gsocorgs/2019/nikhef.md]: There is already a Nikhef logo but with another name (`images/logo_nikhef.svg`)

File: _gsocproposals/2019/proposal_VP2Simulation.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+## Expected results
+Assessment of the performance of a pattern-recognition algorithm that
+has been augmented with the time-of-traversal of particles in terms of
+computational performance and ability to find trajectories.
+
+## Desirable Skills
+  * C++ knowledge
+  * Interest in pattern recognition algorithms
+
+## Mentors
```
Comment (aflp91) [inline _gsocproposals/2019/proposal_VP2Simulation.md]: Please, add mentors in `gsoc/2019/mentors.md` file.

File: _gsocproposals/2019/proposal_VP2Simulation.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+## Description
+
+LHCb [1] is a general purpose forward detector at CERN's Large Hadron
+Collider. To further extend the scientific reach of the experiment, a
+future upgrade is foreseen to be installed in 2030, which is intended
+to be able to cope with a factor of 10 increase in the number of
+proton-proton collisions per second. Studies of the technology for
+this future detector are now underway.
+
+Nikhef [2] is the Dutch institure for sub-atomic physics and it's LHCb
```
Comment (aflp91) [inline _gsocproposals/2019/proposal_VP2Simulation.md]: Typos:
- institure => institute
- it's => its

**Comment by JavierCVilla** [comment]
@roelaaij , what would be the percentages of coding and testing that you expect for this project?

**Comment by roelaaij** [comment]
@JavierCVilla I would expect about 60% coding 20% analysis of computational performance and 20% analysis of physics performance.
I've reworded the "Task Ideas" section to reflect this

**Pull request merged**